recipe/meta.yaml

```diff
@@ -12,13 +12,14 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
+  number: 3
   skip: true  # [win]
   skip: true  # [py2k]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - python
```

Comment:
This needs to be build host, see the tiledb-py feedstock section here:
https://github.com/conda-forge/tiledb-py-feedstock/blob/e2ad49eb6866d48355eb037ee7cc9039f89595c2/recipe/meta.yaml#L18-L22